src/app.ts

```diff
@@ -176,16 +173,74 @@ export async function newApp(
   // Setup routes
   //
 
-  app.get("/login", spidAuth);
-
+  await registerLoginRoute(app);
   registerPublicRoutes(app);
   registerAuthenticationRoutes(app, authenticationBasePath);
   registerAPIRoutes(app, APIBasePath, allowNotifyIPSourceRange);
   registerPagoPARoutes(app, PagoPABasePath, allowPagoPAIPSourceRange);
 
+  const timer = idpMetadataUpdater(
+    app,
+    container.resolve<number>(IDP_REFRESH_INTERVAL_SECONDS) * 1000
```

Comment:
for clarity, assign this value to an intermediate variable

Reply:
fixed in a3bb176